.github/ISSUE_TEMPLATE/config.yml

```
@@ -6,5 +6,5 @@ contact_links:
     about: Please ask questions about Cutter here or one of the other community channels, not in the issue tracker.
 
   - name: Questions IRC
-    url: https://webchat.freenode.net/#cutter
-    about: "#cutter on irc.freenode.net"
\ No newline at end of file
+    url: https://kiwiirc.com/nextclient/#irc:/irc.libera.chat/#cutter
```

I guess we need it in format `irc://...`, right?
```suggestion
    url: https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#cutter
```